kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabTcpIpUi.java

```diff
@@ -338,6 +338,11 @@ public void refresh() {
         }
     }
 
+    @Override
```

Comment:
Please change the header to: " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates"